airbyte-integrations/connectors/source-yandex-direct/source_yandex_direct/schemas/ad_groups.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
It is missing the additionalProperties and usually the fields are `["null", "type"]`

Reply:
I delete this file from PR^ it from another source.